tools/fs/files.ts

```diff
@@ -373,12 +373,12 @@ export const blankHash = "47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=";
 // Returns a base64 SHA256 hash representing a tree on disk. It is not sensitive
 // to modtime, uid/gid, or any permissions bits other than the current-user-exec
 // bit on normal files.
-export function treeHash(root: string, options: {
-  ignore: (path: string) => boolean;
+export function treeHash(root: string, optionsParams: {
```

Comment:
@afrokick can you review my change? I'm not a TS expert but I did the change based on my general knowledge ;) It was breaking with TS2783 error

Reply:
LGTM